Makefile

```diff
@@ -11,11 +11,11 @@ BINDIR= $(shell pwd)/bin
 VPATH = $(SRCDIR)
 
 CFLAGS=-c -g -Wall `root-config --cflags` -I${INCDIR}
-LDFLAGS=`root-config --glibs` -lHistPainter -lMinuit -L${ROOTSYS}/lib
+LDFLAGS=`root-config --glibs` -lHistPainter
 
 TARGET1=field_to_csv.cpp
 TARGET2=ptf_analysis.cpp
-TARGET3=ptf_ttree_analysis.cpp
+TARGET3=temperature_reading.cpp
```

Comment:
what happened to `ptf_ttree_analysis`?

Reply:
I simply, did not wanted to create another target, that is what I will do !